setup_early_access.py

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright 2021-2025 DataRobot, Inc. and its affiliates.
+#
+# All rights reserved.
+#
+# DataRobot, Inc.
+#
+# This is proprietary source code of DataRobot, Inc. and its
+# affiliates.
+#
+# Released under the terms of DataRobot Tool and Utility Agreement.
+from datetime import datetime
+
+from setuptools import find_packages
+from setuptools import setup
+
+from setup_common import DESCRIPTION_TEMPLATE
+from setup_common import common_setup_kwargs
+from setup_common import version
+
+version += datetime.today().strftime(".%Y.%m.%d")
+
+python_versions = ">= 3.9"
+
+classifiers = [
+    "Development Status :: 4 - Beta",
```

Comment:
Nit.
I know we have `Beta` in our DR PySDK's early access release:
https://github.com/datarobot/public_api_client/blob/master/setup_weekly.py#L29

But I wonder if we want to use that naming convention here?
CC @asuslov and @nate-goudreault for your thoughts.

Reply:
Anyway something to think about as we get further along in this project 🤷

Reply:
I would lean towards Preview, if possible. Just to stay consistent with how we describe feature availability.

Reply:
```suggestion
    "Development Status :: 4 - Preview",
```